estelleser/tutorial3/homework3/code/musicstats.sh

```diff
@@ -0,0 +1,310 @@
+#!/bin/bash
```

Comment:
Please remove this file for this submission!

Reply:
Hi, I removed the homework4 folder as you asked.
While your didn't accepted my pull request, I continued to work on homework3 on this branch. Sorry to disturb you.

Reply:
Yes the pull request is kind of a permanent request, if you make changes to a branch that is pending a request they will be reflected in the pull request.
I did not want you to delete the whole homework folder, just your exercise `musicstats.sh`.
However is no big deal, I will accept the pull request anyway so you can proceed forward.